packages/test-ui/src/components/Router.tsx

```diff
@@ -14,9 +14,41 @@ type RouterProps = {
 
 const decimals: Record<string, number> = {};
 
+const ASSETS: Record<number, { [asset: string]: string }> = {
+  56: {
+    usdc: "0x8AC76a51cc950d9822D68b83fE1Ad97B32Cd580d",
+    usdt: "0x55d398326f99059ff775485246999027b3197955",
+    dai: "0x1af3f329e8be154074d8769d1ffa4ee058b1dbc3",
+  },
+  100: {
+    usdc: "0xDDAfbb505ad214D7b80b1f830fcCc89B60fb7A83",
+    usdt: "0x4ecaba5870353805a9f068101a40e0f32ed605c6",
+    dai: "0x0000000000000000000000000000000000000000",
+  },
+  137: {
+    usdc: "0x2791bca1f2de4661ed88a30c99a7a9449aa84174",
+    usdt: "0xc2132d05d31c914a87c6611c10748aeb04b58e8f",
+    dai: "0x8f3cf7ad23cd3cadbd9735aff958023239c6a063",
+  },
+  250: {
+    usdc: "0x04068da6c83afcfa0e13ba15a6696662335d5b75",
+    usdt: "0x049d68029688eabf473097a2fc38ef61633a3c7a",
+    dai: "0x8d11ec38a3eb5e956b052f67da8bdc9bef8abf3e",
+  },
+};
```

Comment:
if we have the names, can we pull this from the `chainData`? may be easier in the future if people want to check for a list of names too

Reply:
is that in chaindata? lol lemme see

Reply:
oh we do, nice. yep will fix